types/defines/rpc.d.ts

```diff
@@ -295,4 +295,18 @@ declare module 'cloudflare:workers' {
   }
 
   export const env: Cloudflare.Env;
+
+  export interface DefaultHandler {
+    fetch?(request: Request): Response | Promise<Response>;
+    tail?(events: TraceItem[]): void | Promise<void>;
+    trace?(traces: TraceItem[]): void | Promise<void>;
+    scheduled?(controller: ScheduledController): void | Promise<void>;
+    queue?(batch: MessageBatch<unknown>): void | Promise<void>;
+    test?(controller: TestController): void | Promise<void>;
```

Comment:
It looks like email is missing.

Could this type be retrieved from existing types to avoid missing fields when a prop is added?

Reply:
I copied the contents of WorkerEntrypoint. So, I'm not sure which is correct.

Reply:
@jasnell I don't understand what you mean.

`nodeCompatHttpServerHandler()` only provides `fetch`, right? How would "have any of our other entrypoints also defined on it optionally"?